extensions/wikia/AdEngine/js/template/bfaaDesktop.js

```diff
@@ -102,8 +105,8 @@ define('ext.wikia.adEngine.template.bfaaDesktop', [
 			runOnReady(iframe, params, videoSettings);
 			wrapper.style.opacity = '';
 
-			if (params.loadMedrecFromBTF) {
-				// refresh after uapContext.setUapId
+			if (!context.opts.disableSra && params.loadMedrecFromBTF) {
```

Comment:
Please, use uber-awesome feature in adContext: https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/context/adContext.js#L288
`adContext.get('opts.disableSra')`